index.html

```diff
@@ -1,9 +1,81 @@
 <!DOCTYPE html>
-<html>
-  <head>
-  </head>
-  <body>
-    <div>
-    </div>
-  </body>
+<html lang = en>
```

Comment:
For element attributes, always use double quotes around the value, and never use spaces around the `=`. For example, here we'd like to see
`<html lang="en">`